application/webapp-service/src/test/java/webapp/tier/service/socket/ServiceSocketTest.java

```diff
@@ -0,0 +1,89 @@
+package webapp.tier.service.socket;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import javax.inject.Inject;
+import javax.websocket.RemoteEndpoint;
+import javax.websocket.Session;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+class ServiceSocketTest {
+
+	@Inject
+	ServiceSocket socket;
+
+	String id = "testOnOpen";
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5948487)